index.js

```diff
@@ -10,6 +10,7 @@ function trace (spec) {
   const {module, vtable, functions} = spec;
   const {onError} = spec.callbacks;
 
+  let listeners = Array();
```

Comment:
```suggestion
  const listeners = [];
```

- Always use `const` when a variable is assigned just once, `let` should only be used when re-assigning is needed.
- Prefer array literals over invoking the constructor.